cds/config.py

```diff
@@ -93,6 +93,10 @@ def _(x):
         'task': 'invenio_accounts.tasks.clean_session_table',
         'schedule': timedelta(days=1),
     },
+    'tasks_status': {
+        'task': 'cds.modules.deposit.tasks.preserve_celery_states_on_db',
+        'schedule': timedelta(days=7),
```

Comment:
I would say yes if we find a way to make the `preserve_celery_states_on_db()` more lightweight.
Do you think there is a way to reduce the number of deposit to update?